cmd/cmd.go

```diff
@@ -494,13 +510,18 @@ func generate(cmd *cobra.Command, model, prompt string, wordWrap bool, format st
 	var currentLineLength int
 	var wordBuffer string
 
-	request := api.GenerateRequest{Model: model, Prompt: prompt, Context: generateContext, Format: format}
+	images := make([]api.ImageData, 0)
+	for _, i := range opts.Images {
+		images = append(images, api.ImageData(i.Data))
```

Comment:
Rather than sending all the image data on every request would it be possible to upload the image to the server than reference it be image ID in the requests? Theres some implications here for data retention lifetime and whatnot, but just an idea.